lib/roberto_barros.rb

```diff
@@ -20,4 +20,30 @@ def self.quote
       "Reboot quem dá é a vida.",
     ].sample
   end
+
+  def self.in_ingrish
+    return [
+      "Ueh?",
+      "Sumo fighters can't wipe their own asses without help.",
+      "I think people who tattoo their faces don't grasp how society works.",
+      "If it is bad, installing Windows will make it worse!",
+      "Neh...",
+      "There is no such thing as simplified project, there is Devs without imagination.",
+      "If it is not simple, it is not Ruby.",
+      "A site is never ready but you need to deliver it at some point.",
+      "And then you get an error in our pretty face!",
```

Comment:
"And then you get an error message in our pretty face!",
```suggestion
      "And then you get an error in our pretty face!",
```